src/include/daos_srv/daos_engine.h

```diff
@@ -36,6 +36,9 @@ extern unsigned int	 dss_tgt_nr;
 /** Storage path (hack) */
 extern const char	*dss_storage_path;
 
+/** Number of storage tiers (hack) */
+extern unsigned int	 dss_storage_tiers;
+
```

Comment:
This is not used anymore? could be removed.

Reply:
thanks